packages/user/src/types/types.ts

```diff
@@ -42,3 +43,19 @@ export interface SignInUpPromise {
 export interface ErrorResponse {
   data: { message: string; status: "ERROR" };
 }
+export interface InvitedByType {
+  email: string;
+  givenName: string;
+  id: string;
+  isActiveUser: boolean;
+  roles: string[];
+  surname: string;
+  username: string;
+}
+export interface User extends UserType {
```

Comment:
Since, we are extending UserType, we don't need to add duplicate properties
This should do the work.

```
export interface ExtendedUser extends UserType {
  appId?: number;
  isActiveUser: boolean;
  invitedBy: UserType & { isActiveUser: boolean;  }
}
```